src/components/Aside/ComingSoon.vue

```diff
@@ -1,24 +1,19 @@
 <template>
-    <article class="border-my-color-pale-gray border-4 rounded-2xl p-3">
-        <div class="flex justify-between">
-            <h2 class="text-yellow-400">Comings</h2>
-            <ul class="flex gap-x-2">
-                <li><a href="">more</a></li>
-            </ul>
-        </div>
-        <div class="grid grid-cols-1 lg:grid-cols-4 md:grid-cols-5 sm:grid-cols-4 gap-4 mt-3 overflow-hidden">
-            <div v-for="movie in data" :key="movie.id" class="relative mx-auto w-full cursor-pointer" aria-selected="true">
-                <img :src="movie.img" alt="" class="w-full lg:h-full rounded-xl  h-28 object-cover">
+    
+            <div class="relative mx-auto w-full cursor-pointer" aria-selected="true">
```

Comment:
why you use aria-selected?

Reply:
in my opinion it helps users with disabilities can understand the current selection